src/image.c

```diff
@@ -251,4 +254,7 @@ _Bool Img_WriteJPEG(const char *path, byte *data, uint32_t width, uint32_t heigh
 
 	fclose(f);
 	return true;
+#else
+	return false;
+#endif
```

Comment:
Yep, that's why I disabled it for now. No sense in me getting jpeglib working if we don't need it